<ui version="4.0" >
 <class>ClientForm</class>
 <widget class="QWidget" name="ClientForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Client Tree</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTreeWidget" name="mp_treeWidget" >
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="actionRefreshClients" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/view-refresh.png</iconset>
   </property>
   <property name="text" >
    <string>Refresh Client List</string>
   </property>
   <property name="statusTip" >
    <string>Requery the director for the list of clients.</string>
   </property>
  </action>
  <action name="actionListJobsofClient" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/emblem-system.png</iconset>
   </property>
   <property name="text" >
    <string>List Jobs of Client</string>
   </property>
   <property name="statusTip" >
    <string>Open a joblist page selecting this client.</string>
   </property>
  </action>
  <action name="actionStatusClientInConsole" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/status-console.png</iconset>
   </property>
   <property name="text" >
    <string>Status Client In Console</string>
   </property>
   <property name="statusTip" >
    <string>Execute status client in console.</string>
   </property>
  </action>
  <action name="actionPurgeJobs" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/weather-severe-alert.png</iconset>
   </property>
   <property name="text" >
    <string>Purge Jobs</string>
   </property>
   <property name="statusTip" >
    <string>Purge jobs peformed from this client.</string>
   </property>
  </action>
  <action name="actionPrune" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/edit-cut.png</iconset>
   </property>
   <property name="text" >
    <string>Prune Jobs</string>
   </property>
   <property name="statusTip" >
    <string>Open the diaolog to prune for this client.</string>
   </property>
  </action>
  <action name="actionStatusClientWindow" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/status.png</iconset>
   </property>
   <property name="text" >
    <string>Status Client Window</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../main.qrc" />
 </resources>
 <connections/>
</ui>
